src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -119,6 +124,44 @@ const StatusConstruct: UiStateDictionary = {
   },
 }
 
+const DownloadConfirmationContent = (props: {
```

Comment:
I pulled this function component out so it's pure and doesn't rely on state in the `DownloadConfirmation` component